opni-insights-service/app/main.py

```diff
@@ -399,3 +400,51 @@ async def index_logs(start_ts: int, end_ts: int):
     except Exception as e:
         # Bad Request
         logging.error(e)
+
+
+@app.get("/areas_of_interest")
+async def get_areas_of_interest(start_ts: int, end_ts: int):
+    logging.info(
+        f"Received request to obtain all areas of interest between {start_ts} and {end_ts}"
+    )
+
+    query = {
+        "query": {
+            "bool": {
+                "must": [{"match": {"anomaly_level.keyword": "Suspicious"}}],
```

Comment:
Why are we just matching for suspicious log messages and not anomalous log messages.

Reply:
Thanks for pointing this out. Was used for development purposes. Fixed for production use case